applications.js

```diff
@@ -1205,7 +1205,12 @@ var CosmicAppsDialog = GObject.registerClass({
 
         const cosmicDock = Main.extensionManager.lookup("cosmic-dock@system76.com");
         if (cosmicDock && cosmicDock.state === ExtensionState.ENABLED) {
-            cosmicDock.stateObj.dockManager._allDocks.forEach((dock) => dock._onOverviewHiding());
+            const dockManager = cosmicDock.stateObj.dockManager
```

Comment:
Under some circumstances `stateObj` can also be null. https://github.com/pop-os/cosmic-workspaces/blob/master_jammy/util.js#L67 checks for it. We should probably do that here too.